lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -1331,8 +1357,670 @@
 
     // Intl.DateTimeFormat, Date.prototype.toLocaleString, Date.prototype.toLocaleDateString, Date.prototype.toLocaleTimeString
     var DateTimeFormat = (function () {
+        if ((InitType === 'Intl' || InitType === 'Date') && isPlatformUsingICU) {
+            const narrowShortLong = ["narrow", "short", "long"];
+            const twoDigitNumeric = ["2-digit", "numeric"];
+            const allOptionValues = [...twoDigitNumeric, ...narrowShortLong]
```

Comment:
I'm not sure whether these types of things are perf-sensitive, but if they are do you know whether this is more effective than `twoDigitNumeric.concat(narrowShortLong)`?

Reply:
No, that is probably better, I thought concat modified the source array. For what its worth, this is not perf-critical code.

Reply:
This has made me realize that Array.prototype[Symbol.iterator] is writable and configurable, so I think that means _any_ function that want to use the default iterator like rest/spread are taintable. Thats a shame.